projects/kit/components/input-date/input-date.component.ts

```diff
@@ -190,8 +196,10 @@ export class TuiInputDateComponent
         return this.interactive && !this.computedMobile;
     }
 
-    get computedMask(): TuiTextMaskOptions {
-        return this.activeItem ? EMPTY_MASK : this.textMaskOptions;
+    get computedMask(): TextMaskConfig {
+        return (this.activeItem
+            ? EMPTY_MASK
+            : this.textMaskOptions) as TuiTextMaskOptions as unknown as TextMaskConfig;
```

Comment:
Why don't you type `textMaskOptions` at the declaration instead?

Reply:
then there will be problems

<img width="499" alt="image" src="https://user-images.githubusercontent.com/12021443/157455448-ef87212e-97f5-4b3e-8638-5a19ed037c91.png">

<img width="698" alt="image" src="https://user-images.githubusercontent.com/12021443/157455474-bd8c30d6-fbf4-4560-aa47-3ea1852ace47.png">

Reply:
You type getter as `TextMaskConfig` and typecast `this.textMaskOptions` at the place where it's defined. It's a readonly prop, let's even move it to a constant so the name is minified.

Reply:
Sorry, I don't understand (( I can't do different manipulations due to type mismatch